js/main.js

```diff
@@ -81,11 +81,11 @@ function handleContainer(switchView) {
     }
   }
 }
-function handleClick(handleViewNavigation) {
+function handleViewNavigation(switchEvent) {
   if (!event.target.getAttribute('data-view')) {
     return;
   }
-  handleContainer(handleViewNavigation);
+  switchView(switchView);
```

Comment:
Passing `switchView` as an argument to `switchView`? Did you test this code before turning it in again?